packages/server/lib/cloud/studio/StudioLifecycleManager.ts

```diff
@@ -99,9 +110,6 @@ export class StudioLifecycleManager {
 
       this.updateStatus('IN_ERROR')
 
-      // Clean up any registered listeners
-      this.listeners = []
```

Comment:
Why was all of the listeners cleanup logic removed?

Reply:
That being said, let's make sure that we don't get duplicate listeners when switching projects in global mode